3/문자열다루기기본.js

```diff
@@ -0,0 +1,10 @@
+function solution(s) {
+  if (s.length == 4 || s.length == 6) {
+    for (let i of s) {
+      if (!("0" <= i && i <= "9")) return false;
+    }
+    return true;
```

Comment:
```suggestion
    let isNumber = /\D/g.test(s);
    if(!isNumber) return true;
    return false;
```
문자열에서 정규표현식이라는 것을 이용하면 for...of문을 사용하지 않고도 특정 문자 패턴이 대응되는지 알 수 있습니다. 
정규표현식 공부해 보시는 것도 좋을 것 같아요🙂
- https://developer.mozilla.org/ko/docs/Web/JavaScript/Guide/Regular_Expressions
- https://www.youtube.com/watch?v=t3M6toIflyQ